README.md

```diff
@@ -1,77 +1,373 @@
+Here are the accounts you will need to store your projects, solve problems, prototype, network, write, and be inspired by others' work. You'll also be able to see some fantastic projects from others.
 
-Another important aspect of communicating your brand on the Internet as a developer is being able to use Markdown formatting. Markdown is a way to write plain text documents that get formatted into html for display on the web. This guide you're currently reading is written in Markdown. Markdown is extremely popular in 2018 and as part of the prework, you're going to take some time to understand it. <br />[Markdown Tutorial](https://www.markdowntutorial.com/)  
-[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
 
-<h2 id="command-line">Know Your Tool</h2>
+#### [LinkedIn For Veterans](https://socialimpact.linkedin.com/programs/veterans/premiumform)
 
-Being able to code well requires you to be able to use your machine well. We're going to start with setting up your machine and then move into learning a few tools that are staples to web development. If you're on Windows your setup instructions are [here](https://github.com/Vets-Who-Code/prework/blob/master/Windows-Tooling.md), MacOS users look [here](https://github.com/Vets-Who-Code/prework/blob/master/MacOS-Tooling.md).
+As a member of the U.S. military community, you can receive one year of LinkedIn Premium through SheerID's collaboration with LinkedIn. This service offers unlimited access to more than 10,000 courses through our LinkedIn Learning platform as a part of the program.
 
-### Code Editor
+LinkedIn Learning is also pleased to offer eligible military spouses one year of access to LinkedIn Premium. As part of the program, you'll also gain access to more than 17,000 courses through LinkedIn Learning to help you manage your career and nurture your professional development.
 
-VS Code is our primary text editor and is **MANDATORY**. Download it [here](https://code.visualstudio.com/download) and play around with it to become familiar since you will spend many hours in this environment!  To help with setup, we have created an extension pack that will get you all of the extensions you will need to handle all of the different formats and configurations during your cohort coursework.  It is available [here](https://marketplace.visualstudio.com/items?itemName=VetsWhoCode.vetswhocode-extension-pack).
+LinkedIn Premium is a subscription-based service that offers users additional features not available on the free LinkedIn platform. These features include seeing who has viewed your profile, sending unlimited InMails to recruiters, and accessing exclusive content and events.
 
-### Command Line Interface
+LinkedIn is a great way to connect with people with similar professional interests and network with potential hiring partners. We also have a LinkedIn Group that we would love for you to join!
```

Comment:
> We also have a LinkedIn Group that we would love for you to join!

This is a Call To Action (CTA), there should be a link to the group here.

Reply:
Copy, will link.